app/models/employee.rb

```diff
@@ -22,6 +22,9 @@ class Employee < ActiveRecord::Base
                   :tax_table_column, :personal, :clearingnumber, :bank_account
 
   WAGE_TYPES = ['Fixed', 'Invoiced']
+  PAYROLL_TAX_55 = '0.3142'
```

Comment:
what does 55, 59 and 61 come from? is there a link to skatteverket.se/somewhere we could add for our future selves?